docs/conf.py

```diff
@@ -0,0 +1,261 @@
+    r"https://www.chef.io/products/chef-infra/",  # Site works but creates SSLError
+    r"https://plonedemo.kitconcept.com",  # Did Not Connect: Potential Security Issue
+    r"https://www.packtpub.com/.*",  # test say 500 Server Error but manually they work
+    r"https://www.dipf.de/.*",  # a timeout from time to time
+    r"https?://plone-conference.localhost.*",
+    # ### Start of list of anchored links
+    # Prior to each PloneConf, uncomment these lines to verify that the links work,
+    # although the anchor cannot be found.
+    # GitHub rewrites anchors with JavaScript.
+    # See https://github.com/plone/training/issues/598#issuecomment-1105168109
+    "https://github.com/collective/awesome-volto#addons",
+    "https://github.com/collective/collective.easyform#collectivez3cformnorobots-support",
+    "https://github.com/collective/collective.easyform#recaptcha-support",
+    "https://github.com/collective/collective.recipe.solrinstance/blob/master/README.rst#multi-core-solr",
+    "https://github.com/nvm-sh/nvm#install-script",
+    "https://github.com/plone/plone.app.contentlisting/#methods-of-contentlistingobjects",
+    "https://github.com/plone/plone.app.contenttypes#changing-the-base-class-for-existing-objects",
+    "https://github.com/plone/plone.recipe.zope2instance#advanced-logging-options-for-wsgi",
+    "https://github.com/repoze/repoze.catalog/blob/master/docs/usage.rst#query-objects",
+    "https://plone.github.io/mockup/dev/#pattern/autotoc",
+    "https://plone.github.io/mockup/dev/#pattern/modal",
+    "https://plone.github.io/mockup/dev/#pattern/moment",
+    "https://github.com/collective/collective.exportimport#faq-tips-and-tricks",
+    "https://github.com/plone/plone.app.contenttypes/tree/2.2.x#migration",
+    "https://robotframework.org/SeleniumLibrary/SeleniumLibrary.html#Keywords",
+    "https://solr.apache.org/guide/8_2/updatehandlers-in-solrconfig.html#UpdateHandlersinSolrConfig-commitWithin",
+    "https://www.npmjs.com/package/axios#example",
+    "https://github.com/plone/plone.restapi/blob/afde2a940d2518e061eb3fe30093093af55e3a50/src/plone/restapi/services/content/configure.zcml#L15-L20",
+    "https://github.com/plone/plone.rest#cors",
+    "https://github.com/plone/plone.docker#for-basic-usage",
+    "https://github.com/nodejs/release#release-schedule",
+    # ### End of list of anchored links
+]
+linkcheck_allowed_redirects = {
+    # All HTTP redirections from the source URI to the canonical URI will be treated as "working".
+    r"https://chrome\.google\.com/webstore/detail/.*": r"https://consent\.google\.com/.*",
+}
+linkcheck_anchors = True
+linkcheck_timeout = 10
+linkcheck_retries = 2
+
+# This is our wordlist with known words, like Github or Plone ...
+spelling_word_list_filename = "spelling_wordlist.txt"
+spelling_ignore_pypi_package_names = True
```

Comment:
Do you want to use Vale for docs instead of or in addition to the Python spellchecker?

Reply:
I don't know? Recommendations?

Reply:
For narrative docs, I would go with Vale. Here's a good example to port. https://github.com/plone/volto/pull/4586/files